content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -139,4 +161,21 @@ For example: to enable deployments and daemonsets, set
 
 {{< note >}}Individual resource enablement/disablement is only supported in the `extensions/v1beta1` API group for legacy reasons.{{< /note >}}
 
+## Persistence
+
+Kubernetes stores its serialized state in terms of the API resources; the default persistence
+store is {{< glossary_tooltip term_id="etcd" >}}. The only API persistence store that
```

Comment:
It's weird to say "the default is X, also X is the only thing supported". I'd rather say "the supported storage backend is X, you'll need to find an adapter if you want to use something else (e.g. https://github.com/rancher/kine).